input/haleu/sensitivity-analysis/oat/lwr_lifetime/lwr_lifetime_driver.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import subprocess
+import dakota.interfacing as di
+import sys
+import os
+from turtle import up
+sys.path.append('../../../../../scripts')
+import dakota_input as inp
+import output_metrics as oup
+import create_AR_DeployInst as cdi
+# import output as oup
+# ----------------------------
+# Parse Dakota parameters file
+# ----------------------------
+
+params, results = di.read_parameters_file()
+
+# -------------------------------
+# Convert and send to Cyclus
+# -------------------------------
+
+# Edit Cyclus input file
+cyclus_template = 'lwr_lifetime_input.xml.in'
+scenario_name = 'lwr_' + str(round(params['lwr']))
+variable_dict = {'handle': scenario_name,
+                 'lwr_lifetime': str(int(params['lwr']))}
+output_xml = './cyclus-files/lwr_lifetime_' + str(params['lwr']) + '.xml'
+inp.render_input(cyclus_template, variable_dict, output_xml)
```

Comment:
This entire block, with some small changes for the particular scenario, gets repeated between `lwr_lifetime_driver.py`, `mmr_share_driver.py`, `transition_start_driver.py`, and xe-100_share_driver.py`. It should be parameterized as a function in its own file and then called from each driver script.